### ISSUE TITLE
Invalid request Validation failed

### ISSUE THREAD

**Issue by exalis86** [Prowlarr/Prowlarr#669]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hello, since the last update Prowlarr can not connect to my tracker

https://yggtorrent.nz/

The site is working and my credentials are correct.
 

### Expected Behavior

I would like Prowlarr to work to do my research with Radarr and Sonarr, like before...

### Environment

```markdown
- OS: openmediavault 5.6.1.20-1 (Usul)
- Prowlarr: 0.1.6.1184
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: Firefox 95.0 (also tested on Chrome, Edge and Brave)
```

### What branch are you running?

Develop

### Trace Logs?

[prowlarr.txt](https://github.com/Prowlarr/Prowlarr/files/7654082/prowlarr.txt)

[AB#2070](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2070)

**Comment by bakerboy448** [comment]
- Info logs are not trace logs.
- your blocked by cloudflare
Please use one of the support channels: [Reddit](https://www.reddit.com/r/prowlarr/) or  [Discord](https://prowlarr.com/discord) for support/questions.
- duplicate of https://github.com/Prowlarr/Prowlarr/issues/660

**Comment by exalis86** [comment]
Sorry but it was working fine before the last update and I have exactly the same error with Flaresolverr.

Problem solved using Jackett. Thanks anyway.
